packages/iris-grid/src/IrisGrid.jsx

```diff
@@ -285,8 +284,14 @@ export class IrisGrid extends Component {
       pendingDataMap,
       canCopy,
       frozenColumns,
+      isSearchBarToggleVisible,
     } = props;
 
+    // Hide search bar is serverConfig.showSearch is false.
```

Comment:
Don't mention `serverConfig.showSearch` here, since this package shouldn't require a serverConfig or anything like that.
We're just disallowing the action if `canToggleSearch` flag is false.

Also - should put a check in the `toggleSearchBar()` action itself, and return if the action isn't available. If you do that, you don't really need to do this part and not register the action, since the keyboard shortcut just won't do anything then.